include/aligator/modelling/multibody/contact-force.hpp

```diff
@@ -18,7 +18,7 @@ template <typename Scalar> struct ContactForceDataTpl;
  */
 
 template <typename _Scalar>
-struct ContactForceResidualTpl : StageFunctionTpl<_Scalar>, frame_api {
```

Comment:
Is the `frame_api` mixin struct still useful after these changes? If not, please mark it as deprecated.